dias/utils/tracker.py

```diff
@@ -262,7 +262,7 @@ def insert_files(self, files):
 
         file_type_rows = self._get_filetypes()
 
-        self.logger.info("Adding {0} to database".format(len(files)))
+        self.logger.debug("Adding {0} to database".format(len(files)))
```

Comment:
Debug makes more sense for this! :)
Can you make the message it self more informative, too?
This is inserting "files" to the database, but not saving any information about them being processed, right? Are these in- or output files of dias?

Reply:
I tried to make them more informative!!

But now as a result...there are more. :grimacing:  :angel: